_posts/2018-02-20-announcing-riff-0-0-4.md

```diff
@@ -12,35 +12,25 @@ We are happy to announce another new release of riff. Thank you, once again, eve
 who contributed to this effort. Here are some of the highlights.
 
 ## a new riff CLI written in go
+The [riff CLI](https://github.com/projectriff/riff-cli/blob/master/docs/riff.md) is now a go binary, available to download from our GitHub [releases](https://github.com/projectriff/riff/releases) page or, if you have a go dev environment, you can install the `riff` command on in your `$GOPATH/bin`.
```

Comment:
"install the `riff` command on in your" should be just "install the `riff` command in your" - remove the extra "on"